src/DynamicsOutputs.jl

```diff
@@ -337,6 +338,64 @@ function (output::OutputStateResolvedScattering1D)(sol, i)
 end
 export OutputStateResolvedScattering1D
 
+"""
+Output the projectile angle from the surface normal and the azimuthal angle for the final image
+"""
+struct OutputScatteringAngle{S,V,T,X}
+    sim::S
+    normal_vector::V
+    incidence_angle::T
+    initial_azimuthal_angle::X
+end
+OutputScatteringAngle(sim; normal_vector=[0, 0, 1], incidence_angle=0., initial_azimuthal_angle=0.) = OutputScatteringAngle(sim, normal_vector, incidence_angle, initial_azimuthal_angle)
+export OutputScatteringAngle
+
+function (output::OutputScatteringAngle)(sol, i)
+    final = last(sol.u) 
+
+    # Scattering angle
+    θₛ = ConfigureAtomic.angle_to_surface_normal(output.sim,
+        DynamicsUtils.get_velocities(final), surface_normal=output.normal_vector)
+
+    # Azimuthal angle
+    θₐ = ConfigureAtomic.get_azimuthal_angle(output.sim,
+    DynamicsUtils.get_velocities(final); incidence_angle=output.incidence_angle, 
+    initial_azimuthal_angle=output.initial_azimuthal_angle) 
+    
+    return [θₛ θₐ]
+end
```

Comment:
The simulation is already accessible from a dynamics run as `sol.prob.p`, which would make it easier to set this output up. 

```suggestion
"""
Output the projectile angle from the surface normal and the azimuthal angle for the final image
"""
struct OutputScatteringAngle{S,V,T,X}
    normal_vector::V
    incidence_angle::T
    initial_azimuthal_angle::X
end
OutputScatteringAngle(sim; normal_vector=[0, 0, 1], incidence_angle=0., initial_azimuthal_angle=0.) = OutputScatteringAngle(sim, normal_vector, incidence_angle, initial_azimuthal_angle)
export OutputScatteringAngle

function (output::OutputScatteringAngle)(sol, i)
    final = last(sol.u) 

    # Scattering angle
    θₛ = ConfigureAtomic.angle_to_surface_normal(sol.prob.p
        DynamicsUtils.get_velocities(final), surface_normal=output.normal_vector)

    # Azimuthal angle
    θₐ = ConfigureAtomic.get_azimuthal_angle(sol.prob.p,
    DynamicsUtils.get_velocities(final); incidence_angle=output.incidence_angle, 
    initial_azimuthal_angle=output.initial_azimuthal_angle) 
    
    return [θₛ θₐ]
end
```